OpenRA.Mods.Common/Activities/Air/ResupplyAircraft.cs

```diff
@@ -48,9 +48,18 @@ protected override void OnFirstRun(Actor self)
 
 		public override Activity Tick(Actor self)
 		{
+			if (ChildActivity != null)
+			{
+				ChildActivity = ActivityUtils.RunActivity(self, ChildActivity);
+				return this;
+			}
+
 			// Conditional fixes being able to stop aircraft from resupplying.
 			if (IsCanceled && NextInQueue == null)
```

Comment:
Won't these conditions be true on the next tick as well? Or does the activity canceling prevent the activity from ticking again?

Reply:
You're right. Changed it again. Hopefully this is now _really_ fixed.